### PR TITLE
Dis 17/comments crud

### PR DESCRIPTION
**Pull request by JSTheGreat**

### CODE REVIEW

File: src/main/java/com/dislinkt/post/model/Comment.java [hunk 1]
```diff
@@ -6,7 +6,6 @@
 public class Comment {
     
     @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Integer id;
```
Comment (rodusek-v) [inline src/main/java/com/dislinkt/post/model/Comment.java:9]: Why did you remove this?

Comment (JSTheGreat) [reply src/main/java/com/dislinkt/post/model/Comment.java:9]: For some reason, it wouldn't allow me to create new entities unless the database was empty upon start.

Comment (rodusek-v) [reply src/main/java/com/dislinkt/post/model/Comment.java:9]: Hmm, ok, lets see about that in next ticket.

**Review by rodusek-v — APPROVED** [review]
No more questions, all good!
